definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -544,19 +549,19 @@ declare module "lodash" {
     // Collection
     countBy<T>(array: $ReadOnlyArray<T>, iteratee?: ?ValueOnlyIteratee<T>): { [string]: number, ... };
     countBy<T>(array: void | null, iteratee?: ?ValueOnlyIteratee<T>): {...};
-    countBy<T: Object>(object: T, iteratee?: ?ValueOnlyIteratee<T>): { [string]: number, ... };
+    countBy<T>(object: ReadOnlyIndexerObject<T>, iteratee?: ?ValueOnlyIteratee<T>): { [string]: number, ... };
     // alias of _.forEach
     each<T>(array: $ReadOnlyArray<T>, iteratee?: ?Iteratee<T>): Array<T>;
     each<T: void | null>(array: T, iteratee?: ?Iteratee<any>): T;
-    each<T: Object>(object: T, iteratee?: ?OIteratee<T>): T;
+    each<A, T: ReadOnlyIndexerObject<A>>(object: T, iteratee?: ?IterateeWithResult<A, T, boolean | void>): T;
```

Comment:
this increases number of type parameters, which can require a change in the code, if they were specified explicitly. 

First option is to remove type T from parameters and specify `ReadOnlyIndexerObject<A>` instead of it in the type definition

Second option is to provide a default value, but that will make type parameter code a bit cumbersome `T: ReadOnlyIndexerObject<A> = ReadOnlyIndexerObject<A>`

But in the end, I've noticed that other existing function already use this style, so I've left it unchanged for now.